README.md

```diff
@@ -16,7 +16,7 @@ To get started with SBT, simply add the following to your `plugins.sbt` file.
 [comment]: # (Start Replace)
 
 ```scala
-addSbtPlugin("io.frees" % "sbt-freestyle" % "0.1.0")
+addSbtPlugin("io.frees" % "sbt-freestyle" % "0.1.3")
 ```
```

Comment:
This is not working because it's missing the "End Replace" tag:
```

[comment]: # (End Replace)
```
Please add it below the `addSbtPlugin` code block (with the new line above). Thanks